sql/2021/accessibility/lighthouse_a11y_score.sql

```diff
@@ -0,0 +1,51 @@
+#standardSQL
+# Percentiles of lighthouse a11y score from 2019 - 2021
+SELECT
+  '2019_07_01' AS date,
+  percentile,
+  APPROX_QUANTILES(score, 1000)[OFFSET(percentile * 10)] AS score
+FROM (
+  SELECT
+    CAST(JSON_EXTRACT(report, '$.categories.accessibility.score') AS NUMERIC) AS score
+  FROM
+    `httparchive.lighthouse.2019_07_01_mobile`),
+  UNNEST([10, 25, 50, 75, 90]) AS percentile
+GROUP BY
+  date,
+  percentile
+
+UNION ALL
+
+SELECT
+  '2020_08_01' AS date,
+  percentile,
+  APPROX_QUANTILES(score, 1000)[OFFSET(percentile * 10)] AS score
+FROM (
+  SELECT
+    CAST(JSON_EXTRACT(report, '$.categories.accessibility.score') AS NUMERIC) AS score
+  FROM
+    `httparchive.lighthouse.2020_08_01_mobile`),
+  UNNEST([10, 25, 50, 75, 90]) AS percentile
+GROUP BY
+  date,
+  percentile
+
+UNION ALL
+
+SELECT
+  '2021_07_01' AS date,
+  percentile,
+  APPROX_QUANTILES(score, 1000)[OFFSET(percentile * 10)] AS score
+FROM (
+  SELECT
+    CAST(JSON_EXTRACT(report, '$.categories.accessibility.score') AS NUMERIC) AS score
+  FROM
+    `httparchive.lighthouse.2021_07_01_mobile`),
+  UNNEST([10, 25, 50, 75, 90]) AS percentile
+GROUP BY
+  date,
+  percentile
+
+ORDER BY
+  date,
+  percentile
```

Comment:
More of a nit than anything but the logic is repeated for each year. I'd recommend restructuring the UNION ALL so that the logic happens after

```suggestion
#standardSQL
# Percentiles of lighthouse a11y score from 2019 - 2021
WITH lighthouse AS (
  SELECT
    '2019_07_01' AS date,
    *
  FROM
    `httparchive.lighthouse.2019_07_01_mobile`
UNION ALL
  SELECT
    '2020_08_01' AS date,
    *
  FROM
    `httparchive.lighthouse.2020_08_01_mobile`
UNION ALL
  SELECT
    '2021_07_01' AS date,
    *
  FROM
    `httparchive.lighthouse.2021_07_01_mobile`
)

SELECT
  date,
  percentile,
  APPROX_QUANTILES(score, 1000)[OFFSET(percentile * 10)] AS score
FROM (
  SELECT
    date,
    CAST(JSON_EXTRACT(report, '$.categories.accessibility.score') AS NUMERIC) AS score
  FROM
    lighthouse),
  UNNEST([10, 25, 50, 75, 90]) AS percentile
GROUP BY
  date,
  percentile
```